lib/python/rose/macro.py

```diff
@@ -236,7 +238,7 @@ def _sorter(self, rep1, rep2):
         id1 = self._get_id_from_section_option(rep1.section, rep1.option)
         id2 = self._get_id_from_section_option(rep2.section, rep2.option)
         if id1 == id2:
-            return cmp(rep1.value, rep2.value)
+            return (rep1.value > rep2.value) - (rep1.value < rep2.value)
```

Comment:
Boolean subtraction?

Reply:
See earlier comments - file added to #2307